src/main/java/school/hei/patrimoine/visualisation/swing/ihm/google/NamedLink.java

```diff
@@ -0,0 +1,11 @@
+package school.hei.patrimoine.visualisation.swing.ihm.google;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+@AllArgsConstructor
+@Getter
+public class NamedLink {
+  private String name;
+  private String value;
+}
```

Comment:
public record NamedLink(String name, String value)` favor records because they are immutable and suit a basic POJO with pre-made constructor, getter, toString, equals and hashcode

Reply:
done